controllers/applicationsnapshotenvironmentbinding_controller.go

```diff
@@ -59,12 +47,9 @@ type SnapshotEnvironmentBindingReconciler struct {
 	Scheme            *runtime.Scheme
 	Log               logr.Logger
 	AppFS             afero.Afero
-	Generator         gitopsgen.Generator
 	GitHubTokenClient github.GitHubToken
 }
 
-const asebName = "SnapshotEnvironmentBinding"
```

Comment:
Should we remove this entire controller?

Reply:
Handled under DEVHAS-640, which @thepetk is owning.